raw/smb-admin/SMBServerTasks.xml

```diff
@@ -0,0 +1,261 @@
+default nfs=()
+mygroup smb=()
+zfs nfs=() smb=()
+     zfs/sandbox/fs1 smb=()
+           sandbox_fs1=/sandbox/fs1</screen><para>The following commands create another file system in the <filename>sandbox</filename> pool called <filename>fs2</filename> and associate that file system with the <filename>myshare</filename> resource name:</para><screen># <userinput>zfs create -o casesensitivity=mixed -o nbmand=on sandbox/fs2</userinput>
+# <userinput>zfs set sharesmb=name=myshare sandbox/fs2</userinput></screen><para>Use the <command>sharemgr show -vp</command> command to verify how the file systems are shared.</para><screen># <userinput>sharemgr show -vp</userinput>
+default nfs=()
+mygroup smb=()
+zfs nfs=() smb=()
+     zfs/sandbox/fs1 smb=()
+           sandbox_fs1=/sandbox/fs1
+     zfs/sandbox/fs2 smb=()
+           myshare=/sandbox/fs2</screen><para>The following command creates a sub file system of <filename>sandbox/fs2</filename> called <filename>sandbox/fs2/fs2_sub1</filename>:</para><screen># <userinput>zfs create -o casesensitivity=mixed -o nbmand=on sandbox/fs2/fs2_sub1</userinput></screen><para>This new file system inherits part of its resource name from its parent and also inherits sharing over SMB, if enabled. Because the resource name for <filename>sandbox/fs2</filename> is <filename>myshare</filename>, the resource name for <filename>sandbox/fs2/fs2_sub1</filename> is <filename>myshare_fs2_sub1</filename>.</para><screen># <userinput>sharemgr show -vp</userinput>
+default nfs=()
+mygroup smb=()
+zfs nfs=() smb=()
+     zfs/sandbox/fs1 smb=()
+           sandbox_fs1=/sandbox/fs1
+     zfs/sandbox/fs2 smb=()
+           myshare=/sandbox/fs2
+           myshare_fs2_sub1=/sandbox/fs2/fs2_sub1</screen><para>If you disable SMB sharing for <filename>sandbox/fs2</filename>, that file system and its children are affected.</para><screen># <userinput>zfs set sharesmb=off sandbox/fs2</userinput>
+# <userinput>sharemgr show -vp</userinput>
+default nfs=()
+mygroup smb=()
+zfs nfs=() smb=()
+     zfs/sandbox/fs1 smb=()
+           sandbox_fs1=/sandbox/fs1</screen><para>The <command>sharemgr show -vp</command> output shows that the <filename>sandbox/fs2</filename> file system and its children are no longer shared over SMB.</para>
+</listitem>
+</itemizedlist>
+</example>
```

Comment:
Something about these examples looks like it's not rendering correctly. The text commands look off and the alignment is too at least for me.

Reply:
I tried to leave the formatting pretty much as it was.  I though it looked OK, but then I'm not really sure what it should look like.

For some of this formatting stuff it might be easier if you could show me somehow.
Maybe setup a google meet call and share your screen or something?